docs/development-update-22.44.md

```diff
@@ -0,0 +1,120 @@
+# Development Update 22.44
+
+## Overview
+
+This is an interim update ("code drop") to give the validation team
+an opportunity to take the DPDK code out for a test drive.
+
+## Project Name
+
+Note that the P4-OVS Split Architecture has a new name: **P4 Control Plane**.
+
+## What's Changed
+
+### Highlights
+
+- Restructured Stratum to support TDI variants
+- Integrated DPDK support from P4-OVS
+- Disabled unsupported port attributes in DPDK
+- Updated OVS to newer release
+- Replace `gnmi_cli` with `gnmi-ctl` in DPDK build
+
+### TDI variants
+
+When we compared the Stratum files in P4-OVS with their Tofino equivalents,
+we found significant changes in the ChassisManager, Hal, Main, and Switch
+classes, as well as SdeWrapper and YangParseTreePaths.
+
+We addressed the first four classes by creating *dpdk* and *tofino*
+subdirectories under the *tdi* directories, and giving each target its
```

Comment:
instead of specifying just tdi here, should specify first that this is under stratum/hal/lib and stratum/hal/bin?

Reply:
I wrote it that way originally and decided that it was an unnecessary detail.